config/logstash.yml

```diff
@@ -379,7 +379,7 @@
 #xpack.management.elasticsearch.sniffing: false
 #xpack.management.logstash.poll_interval: 5s
 
-# X-Pack GeoIP plugin
+# X-Pack GeoIP Database Management
 # https://www.elastic.co/guide/en/logstash/current/plugins-filters-geoip.html#plugins-filters-geoip-manage_update
-#xpack.geoip.download.endpoint: "https://geoip.elastic.co/v1/database"
 #xpack.geoip.downloader.enabled: true
+#xpack.geoip.downloader.endpoint: "https://geoip.elastic.co/v1/database"
```

Comment:
Yes, intentionally:

~~~ diff
-#xpack.geoip.download.endpoint: "https://geoip.elastic.co/v1/database"
+#xpack.geoip.downloader.endpoint: "https://geoip.elastic.co/v1/database"
~~~

To align with naming internally and with the similar feature in Elasticsearch's Geoip database manager. The deprecated pathway is included in `geoip_database_management/extension.rb`.

Reply:
we can add poll interval here
#xpack.geoip.downloader.poll.interval: 24h

Reply:
```suggestion
#xpack.geoip.downloader.endpoint: "https://geoip.elastic.co/v1/database"
#xpack.geoip.downloader.poll.interval: 24h
```

Reply:
I didn't add the poll interval intentionally, and would prefer deferring that discussion until later. I found the poll interval very helpful in manual testing, but there are some very sharp edges (like our `TimeValue` setting not supporting upper/lower bounds on the total value or on granularity) that limit the value of exposing this to users.